### DISCUSSION TITLE
Sending power measurements to inverter

### DISCUSSION THREAD

**Discussion by kroeckx** [wills106/homeassistant-solax-modbus#777] [General]
There is currently a device that converts the power measurements from the power meter to the Solis inverter. Looking at the manual, this looks like it's done over RS-485.

I have the same power measurements in homeassistant. Will homeassistant sent those measurements to the converter?

**Comment by Tomk211** [comment]
I have the same question. Any idea how to manipulate the demanded power. My usecase is that there is a second meter before the q cells smart meter. Meter cascade for separate measuring of heat pump current.

**Comment by kroeckx** [comment]
The reason I would need this is that I also have a battery, and it needs to know such things so it doesn't send all my battery power to the net.
